src/uct/ib/mlx5/exp/ib_exp.h

```diff
@@ -11,7 +11,7 @@
 #  include "config.h"
 #endif
 
-#if HAVE_MLX5_HW && HAVE_VERBS_EXP_H
+#if HAVE_MLX5_HW && defined (HAVE_VERBS_EXP_H)
```

Comment:
Why use two different evaluations? Imo better use `defined` for both

Reply:
unified